test/Makefile

```diff
@@ -0,0 +1,32 @@
+# Fishtape test runner for Nix packages
```

Comment:
could we integrate this file with the parent makefile.. and just have a make test